book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Kindly add `:rentals` to `attr_accessor` since user can read and write rentals. See suggestion below:

```suggestion
  attr_accessor :title, :author, :rentals
```